tests/e2e/pageobjects/openshift/OcpMainPage.ts

```diff
@@ -70,9 +75,8 @@ export class OcpMainPage {
     }
 
     async openImportFromGitPage(): Promise<OcpImportFromGitPage> {
-        await this.waitOpenMainPage();
-        await this.clickOnSelectRoleButton();
-        await this.selectDeveloperRole();
+        Logger.debug(`${this.constructor.name}.${this.openImportFromGitPage.name}`);
```

Comment:
What's the purpose of this logger message? Maybe adding some context to the output would make it more usable. Or should it be removed altogether?

Reply:
At the same time you could consider simpler approach https://github.com/eclipse/che/pull/22201/files#r1189665731

Reply:
I see, that's a really good idea, thx Maryna